### ISSUE TITLE
[DEVEL] Unable to install contracts lib with npm

### ISSUE THREAD

**Issue by d0x2f** [smartcontractkit/chainlink#5663]
**Description**
The readme provides a command to install the contracts with npm but the preinstall script in package.json explicitly fails if yarn isn't used, is this intended?

Relevant line: https://github.com/smartcontractkit/chainlink/blob/a520ef903f5b6c174f519657a909b245c95322ef/contracts/package.json#L9

**Your Environment**
Arch linux
```
$ npm -v
8.1.3
$ node -v
v17.1.0
```

**Steps to Reproduce**
```
$ mkdir test
$ cd test
$ npm install @chainlink/contracts --save
npm ERR! code 1
npm ERR! path /tmp/test/node_modules/@chainlink/contracts
npm ERR! command failed
npm ERR! command sh -c node -e "if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('Please use yarn to install dependencies\n\n')"
npm ERR! [eval]:1
npm ERR! if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('Please use yarn to install dependencies\n\n')
npm ERR!                                                     ^
npm ERR! 
npm ERR! Error: Please use yarn to install dependencies
npm ERR! 
npm ERR! 
npm ERR!     at [eval]:1:59
npm ERR!     at Script.runInThisContext (node:vm:129:12)
npm ERR!     at Object.runInThisContext (node:vm:305:38)
npm ERR!     at node:internal/process/execution:75:19
npm ERR!     at [eval]-wrapper:6:22
npm ERR!     at evalScript (node:internal/process/execution:74:60)
npm ERR!     at node:internal/main/eval_string:27:3
npm ERR! 
npm ERR! Node.js v17.1.0

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dylan/.npm/_logs/2021-12-10T20_01_36_534Z-debug.log
```

**Comment by josteph** [comment]
If this commit https://github.com/smartcontractkit/chainlink/commit/560aa67a4b4740ecf8853612435e9a427efbc256 was intended for development purpose, please use `engine` field instead.

This kinda blocks deployment

**Comment by anh194** [comment]
Same, still cant install chainlink through npm

**Comment by wmitsuda** [comment]
Installing contracts 0.3.0 is completely broken on npm, got the dependabot alert to update it, had to revert because of this issue.

**Comment by wmitsuda** [comment]
BTW, 0.2.2 is fine, 0.3.0 which was just released is broken.

**Comment by the-emerald** [comment]
+1. Cannot use 0.3.0 because `hardhat` uses npm and not yarn.

**Comment by pappas999** [comment]
thanks, can confirm that the 0.3.0 contracts won't install via NPM. I'll check with the team if this is intentional or not, and if it is, will get them to update all relevant documentation

**Comment by scorpion9979** [comment]
I'm experiencing the same issue.

**Comment by TrejGun** [comment]
What a shame! no reaction for bug report after 3 weeks of completely broken package

**Comment by ubudragon** [comment]
wtf, dudes?

**Comment by prasad-kumkar** [comment]
same same

**Comment by Ekzer** [comment]
Same

**Comment by se3000** [comment]
This has been fixed with [version 0.3.1](https://github.com/smartcontractkit/chainlink/pull/5753/). Apologies for the delay, this slipped through the cracks around the holidays. We're going to revisit our process for escalating issues here to ensure this doesn't happen again. Thanks for your patience!
